platform_disk_api/schema.py

```diff
@@ -20,6 +20,7 @@ class DiskRequestSchema(Schema):
         required=False,
         allow_none=True,
     )
+    project_name = fields.String(required=True)
```

Comment:
Won't `required=True` break backward-compatibility? I see that in the handler we use `.get()` to read the value.

Reply:
shouldn't break, api handler adds project_name if it isn't specified before validation. There is test for this https://github.com/neuro-inc/platform-disk-api/blob/iz/support-projects/tests/integration/test_api.py#L237

Reply:
👍